test/setup/Dockerfile.akashtest

```diff
@@ -0,0 +1,14 @@
+# https://hub.docker.com/r/ovrclk/akash/tags?page=1&ordering=last_updated
+FROM ovrclk/akash:0.10.0
```

Comment:
```suggestion
FROM ovrclk/akash:0.10.2
```

Reply:
I don't think they have a docker image for that version